src/main/java/sirius/biz/web/BizController.java

```diff
@@ -141,6 +141,11 @@ protected String getBaseUrl() {
                 LOG.WARN("product.baseUrl is not filled. Please update the system configuration!");
             }
         }
+
+        if (UserContext.getSettings().getConfig().hasPath("tenant.baseUrl")) {
+            return UserContext.getSettings().get("tenant.baseUrl").asString();
```

Comment:
```suggestion
            return UserContext.getSettings().get("tenant.baseUrl").asString(baseUrl);
```

and probably skip if